### DISCUSSION TITLE
Is it possible to pass glow into a terminal command?

### DISCUSSION THREAD

**Discussion by guttermonk** [charmbracelet/glow#636] [General]
Trying to set a key-bind to open a specific md file in glow: 
`kitty -e glow ~/Downloads/artichokes.md`

When I run `glow ~/Downloads/artichokes.md` it opens fine. When I run the above command, the terminal tries to open and closes right away with no error message. What am I doing wrong?

**Comment by Delta456** [comment]
Maybe try ` kitty -e "glow ~/Downloads/artichokes.md"`

**Comment by guttermonk** [comment]
Thank you for the suggestion. That gives me the following error:
```
Failed to launch child: glow ~/Downloads/artichokes.md
With error: No such file or directory
Press Enter or Esc to exit
```

I've confirmed that the file is in fact there, I'm able to open it with an older renderer called mdr, as follows:
`kitty -e mdr ~/Downloads/artichokes.md`

**Reply by Delta456** [reply]
I believe this is a kitty issue. No problem with `glow`.

**Reply by meowgorithm** [reply]
Maybe try `$HOME` instead of `~`? iirc, the tilde is specific to Bash.

**Reply by guttermonk** [reply]
I get the same behavior when I use Alacritty instead of Kitty, and when I use `$HOME` instead of `~`. Good guesses though.

**Reply by Delta456** [reply]
> Maybe try `$HOME` instead of `~`? iirc, the tilde is specific to Bash.

tilde works on zsh too
